common/config/rush/common-versions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/common-versions.schema.json",
+  "preferredVersions": {
+    "follow-redirects": "~1.15.4"
```

Comment:
This repo has dependency on `wait-on`, latest `wait-on` release depends on `axios: 1.6.1.`, this older version of axios has `follow-redirects: 1.15.0` as dependency. Hence, this addition here.

Plus, if `wait-on` is bumped to latest version on this repo, that would be a major version change and I didn't want to break anything in case the bump came with big breaking changes or anything like that, specially since I don't have any knowledge in this repo or the packages here. I can explore that route if you'd like me to though, lmk. The `wait-on` version being used in this repo depends on `axios: 0.25.0` btw :/

Reply:
`wait-on` is only used when launching tests if we need to launch a local storage as well (like minio or azurite). It would be sufficient to check if `storage/minio` `test:integration:backend` launches tests to verify that it didn't break anything.

Reply:
Made the updates, except I am on Mac and it seems the pkg is not supported for MacOS:
<img width="1179" alt="image" src="https://github.com/iTwin/object-storage/assets/98850418/55582185-eb30-4533-b4e7-67147f7e9cd4">

Any suggestions @Animagne ?

Reply:
I'll verify these changes shortly

Reply:
This works fine on my local machine

Reply:
Ok, sounds good, in that case if everything else seems fine, you can go ahead and approve. I don't really need to run the tests locally myself

Reply:
Oops, it seems the Build and Test step on the build is failing now. Let me know what might be causing it, thanks